src/pages/Homepage/iwd/IWDHomepage.vue

```diff
@@ -0,0 +1,813 @@
+<template>
```

Comment:
Majority of work is in this file

Reply:
As mentioned below... you'll want to remove the www-page component here and just keep the slot content. We'll have to do the same for the current homepage moving it's slot-content into it's own file (let's call it defaultHomepage).

Reply:
Again all good for now.